flight/PiOS/Common/pios_rcvr.c

```diff
@@ -34,23 +33,6 @@ static struct pios_rcvr_dev * PIOS_RCVR_alloc(void)
   rcvr_dev->magic = PIOS_RCVR_DEV_MAGIC;
   return(rcvr_dev);
 }
-#else
-static struct pios_rcvr_dev pios_rcvr_devs[PIOS_RCVR_MAX_DEVS];
-static uint8_t pios_rcvr_num_devs;
-static struct pios_rcvr_dev * PIOS_RCVR_alloc(void)
-{
-  struct pios_rcvr_dev * rcvr_dev;
-
-  if (pios_rcvr_num_devs >= PIOS_RCVR_MAX_DEVS) {
-    return (NULL);
-  }
-
-  rcvr_dev = &pios_rcvr_devs[pios_rcvr_num_devs++];
-  rcvr_dev->magic = PIOS_RCVR_DEV_MAGIC;
-
-  return (rcvr_dev);
-}
-#endif
```

Comment:
There already was a heap implementation that uses an array from SHEAP (end of BSS) to end of memory...  So this stuff had no purpose except to waste memory.